src/NServiceBus.Testing.Fakes/TimeoutMessage.cs

```diff
@@ -1,25 +0,0 @@
-﻿namespace NServiceBus.Testing
-{
-    using System;
-
-    /// <summary>
-    /// Represents a message that is timed out, if not processed within a given timespan. Contains the message itself and its
-    /// associated options.
-    /// </summary>
-    /// <typeparam name="TMessage">The message type</typeparam>
-    public class TimeoutMessage<TMessage> : OutgoingMessage<TMessage, SendOptions>
```

Comment:
this is only used by the NServiceBus.Testing `TestingContext` class and should live in that repository.